spec.md

```diff
@@ -463,8 +463,7 @@ values include `None`, Booleans, numbers, and strings, and tuples
 composed from hashable values.  Most mutable values, such as lists,
 and dictionaries, are not hashable, unless they are frozen.
 Attempting to use a non-hashable value as a key in a dictionary
-results in a dynamic error, as does passing one to the built-in
-`hash` function.
+results in a dynamic error.
```

Comment:
I think the old text was correct (does that mean this change needs to flow in the other direction?).

Both `hash` implementations accept only string, not list, say.

Reply:
I think that's correct.

The old text said it's an error to pass a non-hashable value to hash. This might be technically true, but that's irrelevant since hash accepts only strings. No need to mention `hash` here.

Reply:
Oh, of course.